docs/design/simple-rolling-update.md

```diff
@@ -0,0 +1,84 @@
+## Simple rolling update
+This is a lightweight design document for simple rolling update in ```kubectl```
+
+### Lightweight rollout
+Assume that we have a current replication controller named ```foo``` and it is running image ```image:v1```
+
+```kubectl rolling-update rc foo [foo-v2] --image=myimage:v2```
+
+If the user doesn't specify a name for the 'next' controller, then the 'next' controller is renamed to
```

Comment:
We also need to add/choose a label key/value to use as the uniquifying label. As discussed in https://github.com/GoogleCloudPlatform/kubernetes/issues/1743#issuecomment-93774833, I suggest the key `deployment` by default, but the user should be able to override that via flag. Even if the object name is changed back to the original, the label value should still advance.